lib/JsonpMainTemplatePlugin.js

```diff
@@ -18,7 +18,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 				"var installedChunks = {",
 				this.indent(
 					chunk.ids.map(function(id) {
-						return id + ": 0";
+						return JSON.stringify(id) + ": 0";
```

Comment:
Might be good to have a helper that avoids quoting if it's a string and valid identifier; but this would be done by any minifier anyway.